src/uu/du/src/du.rs

```diff
@@ -565,17 +654,19 @@ Try '{} --help' for more information.",
                 let (_, len) = iter.size_hint();
                 let len = len.unwrap();
                 for (index, stat) in iter.enumerate() {
-                    let size = if matches.opt_present("apparent-size") || matches.opt_present("b") {
+                    let size = if matches.is_present(options::APPARENT_SIZE)
+                        || matches.is_present(options::BYTES)
+                    {
                         stat.size
                     } else {
                         // C's stat is such that each block is assume to be 512 bytes
                         // See: http://linux.die.net/man/2/stat
                         stat.blocks * 512
                     };
-                    if matches.opt_present("time") {
+                    if matches.is_present(options::TIME) {
                         let tm = {
                             let secs = {
-                                match matches.opt_str("time") {
+                                match matches.value_of(options::TIME) {
                                     Some(s) => match &s[..] {
```

Comment:
Can you fix the clippy warnings too?
See: https://rust-lang.github.io/rust-clippy/master/#redundant_slicing

Reply:
Fixed :+1: